codersdk/name.go

```diff
@@ -1,4 +1,4 @@
-package httpapi
+package codersdk
```

Comment:
The comment on NameValid in this file needs to be updated because it's not for any name, only for any name without it's own validation function